content/en/guides/core/automations/automation-events.md

```diff
@@ -122,7 +122,11 @@ For example, trigger an automation when average `loss` decreases by at least `.2
 #### Run filters
 This section describes how the automation selects runs to evaluate.
 
-- By default, any run in the project triggers the automation when the event occurs. To consider only specific runs, specify a run filter.
+- By default, any run in the project triggers the automation when the event occurs. To consider only specific runs, specify one or more run filters:
```

Comment:
This intentionally doesn't use the include because it's laid out a bit differently